nupic/algorithms/anomaly.py

```diff
@@ -167,10 +178,35 @@ def __str__(self):
   def __setstate__(self, state):
     """deserialization"""
     self.__dict__.update(state)
-
     if not hasattr(self, '_mode'):
       self._mode = Anomaly.MODE_PURE
     if not hasattr(self, '_movingAverage'):
       self._movingAverage = None
     if not hasattr(self, '_binaryThreshold'):
       self._binaryThreshold = None
+
+
+  def getComputeFn(self, mode):
+    """
+    assing compute() function to be used; 
+    all these functions must take these arguments:
+     activeColumns, prevPredictedColumns, inputValue=None, timestamp=None
+    and return anomaly score
+
+    @param mode   if string: we assign from Anomaly.MODE_* defined functions
+                  if func: we will use that function to compute anomaly score
+    @return a function that is used for anomaly computation
```

Comment:
Better to say callable instead of function. Also, describe the inputs and outputs of this callable.